### PR TITLE
Render data with props and map

### PR DESCRIPTION
**Pull request by mbosselmann**
Render data in data.json for cards with map method and added a key for correct rendering. Also change some changes in footer.css, deleted not needed consts and made a mistake in data.json, which I corrected.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Slytherin Wanted List</title>
+    <title>Hogwarts Wanted</title>
```
Comment (macfrei) [inline public/index.html:27]: Uhh, interesting! 🤩

File: src/components/app/App.js [hunk 2]
```diff
@@ -3,67 +3,27 @@ import "./App.css";
...
+            gender={character.gender}
+            birthday={character.dateOfBirth}
+            patronus={character.patronus}
+            wandWood={character.wand.wood}
+            wandCore={character.wand.core}
+            wandLength={character.wand.length}
+            hogwartsHouse={character.house}
+            imageURL={character.image}
+            key={character.name}
+          />
```
Comment (macfrei) [inline src/components/app/App.js:25]: Looks good! Nice job 👍

File: src/components/card/Card.css [hunk 3]
```diff
@@ -104,6 +104,6 @@
   .section-card {
     width: 500px;
     position: relative;
-    margin: 0 auto;
+    margin: 15px auto 15px auto;
```
Comment (macfrei) [inline src/components/card/Card.css:107]: Should be the same as `15px auto` no?

File: src/components/footer/Footer.js [hunk 4]
```diff
@@ -1,18 +1,13 @@
...
   return (
     <footer>
       <nav id="house-filter">
-        <button className="nav-button-gryffindor">{gryffindor}</button>
-        <button className="nav-button-ravenclaw">{ravenclaw}</button>
-        <button className="nav-button-hufflepuff">{hufflepuff}</button>
-        <button className="nav-button-slytherin">{slytherin}</button>
+        <button className="nav-button-gryffindor">G</button>
+        <button className="nav-button-ravenclaw">R</button>
+        <button className="nav-button-hufflepuff">H</button>
```
Comment (macfrei) [inline src/components/footer/Footer.js:9]: Maybe add `aria-label`s here? Maybe `G`, `R`, `H` are a bit too short to understand right away?

**Review by macfrei — COMMENTED** [review]
Nice job! 💯

File: src/components/footer/Footer.js (hunk 4 above)
Comment (mbosselmann) [reply src/components/footer/Footer.js:9]: Thank you! ❤️  Changed it and you will find a navbar with aria-labels in my next pull request. Wasn't sure about appropriate names. Can you maybe check it and give me feedback?

File: src/components/card/Card.css (hunk 3 above)
Comment (mbosselmann) [reply src/components/card/Card.css:107]: You are right! Changed the `margin` to `15px auto`. :)

File: src/components/footer/Footer.js (hunk 4 above)
Comment (macfrei) [reply src/components/footer/Footer.js:9]: I will! .... tomorrow though 😉

Comment (mbosselmann) [reply src/components/footer/Footer.js:9]: No hurry! I have to create the pull request first. I look forward to your feedback tomorrow. :-)

**Pull request merged**